packages/top-nav/stories/top-nav.stories.ts

```diff
@@ -19,7 +19,8 @@ import '@spectrum-web-components/menu/sp-menu.js';
 import '@spectrum-web-components/menu/sp-menu-item.js';
 import '@spectrum-web-components/menu/sp-menu-divider.js';
 import '@spectrum-web-components/icon/sp-icon.js';
-import { SettingsIcon } from '@spectrum-web-components/icons-workflow';
+import { SettingsIcon, DownloadIcon, LinkIcon, SearchIcon } from '@spectrum-web-components/icons-workflow';
+import { logo, avatar } from './images';
```

Comment:
You've added a handful of new elements to these stories, while the Storybook build handles that without error, we should include their registration modules above for more precise documentation of the usage here.

Reply:
Good call. I struggle with tracking custom element dependencies generally, since they don't throw or anything if used before they're defined.

Reply:
This is great feedback. I've seen some options recently about codifying TS linting in the tsconfig.json that might be worth following up on separately to help ensure that things like `lit-plugin` apply by default (at least to VS Code), as this can be an important situation to overcome.